Automattic/wp-calypso
# Update/whats new styling

#### Proposed Changes

This PR applies the "What's New" modal redesign: p3jbtC4W1tDu7z5uhvtry5-fi-1668%3A72563

| Before  | After |
| ------------- | ------------- |
|   <img width="728" alt="Screenshot 2022-06-27 at 13 12 04" src="https://user-images.githubusercontent.com/7000684/175929316-4bb58d93-836e-4eae-a781-cfb6d208712f.png"> |  <img width="755" alt="Screenshot 2022-06-27 at 13 10 48" src="https://user-images.githubusercontent.com/7000684/175929376-0bdd28c2-78c7-4707-85e2-04ab6297acb9.png"> |


#### Testing Instructions
- Checkout this branch
- Go to the editing-toolkit folder
- Run `yarn dev --sync`
- Compare the modal with the figma designs p3jbtC4W1tDu7z5uhvtry5-fi-1668%3A72563

A short summary of the changes required:
- Modal should be 744px 400px
- Modal should have 4px border radius
- Left content should have 32px padding
- Header should use Recoleta with `$font-title-large`
- Content should use $font-body with `var( --studio-gray-80 );`
- Learn more button should use `var( --studio-blue-50 );`
- Pagination dots should be 6px with 8px spacing. 
- Pagination dots should use `var(--studio-gray-10)` for not selected and `var( --studio-gray-80 )` for the current. 
- The "Next" button should be named "Done" when the guide is on the last page
- The right side should have `var( --studio-blue-50 )` as background
- There should be a modal close button with `var( --studio-white )` color

#### Leftovers
- We can't properly target the modal overlay to adjust the opacity
- We can't change the button "Previous" to have the text "Back"
- The modal will [autofocus on the first element](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/guide/index.js#L45). If the modal's close button is enabled then it will get the focus by default. To mitigate this PR introduces a hack, to add a new close button.  This should be removed and the focus should be on the "Next" button by default. This will be done by extracting the `Guide` to a component that better suits the needs of the new design.

Related to #64817
